tedana/tests/test_external_metrics.py

```diff
@@ -186,14 +186,15 @@ def test_validate_extern_regress_succeeds(caplog):
 
     external_regressors, n_vols = sample_external_regressors()
     external_regressor_config = sample_external_regressor_config()
-    external_regressor_config_expanded = external.validate_extern_regress(
+    external_regressors, external_regressor_config_expanded = external.validate_extern_regress(
         external_regressors=external_regressors,
         external_regressor_config=external_regressor_config,
         n_vols=n_vols,
         dummy_scans=0,
     )
 
     # The regex patterns should have been replaced with the full names of the regressors
+    assert external_regressors.notna().all(axis=None)
```

Comment:
I don't think there are any NaNs in this test, but if you replace the dt columns' first rows (currently zeroes) with NaNs in the test TSV file, that should work.